### ISSUE TITLE
Insert all stackable

### ISSUE THREAD

**Issue by chrstffr** [Roundaround/mc-fabric-inventory-management#15]
I would love to see a shift-"transfer all" that moves all items that matches the contents of a container. 

Example:
Say you have a shulkerbox with 3 and some stacks of dirt, shift-clicking transfer-all would try to move everything from your own inventory that match the contents of the shulkerbox - in this case all the dirt.

**Comment by Roundaround** [comment]
We do already have a similar behavior to this with the auto stack button. The idea behind it is that if you want to make a shulker for dirt, you would just fill it with one dirt per slot. Then every time you come to plop more in, you just use the auto stack button and it'll fill any non-full stacks

**Comment by chrstffr** [comment]
Yeah, I noticed that, but I couldn't figure out a usecase for that. If you have a shulker and want to move all the dirt you justed dug, it's easier to just shift-double-click the dirt than to prep certain slots with 1 dirt per slot, then click the auto-stack-button. Or am I missing something? 🤔
